scoresheets/CleanUp.tex

```diff
@@ -0,0 +1,25 @@
+
+The maximum time for this test is 5 minutes.
+
+\begin{scorelist}
+	\scoreheading{Main Goal}
+	\scoreitem[5]{100}{Place an object at the appropriate location}
+	\penaltyitem[5]{-30}{Receiving human help (telling where an object can be found)}
+	\penaltyitem[5]{-40}{Receiving human help (pointing out object to be moved)}
+	\penaltyitem[5]{-20}{Receiving human help (Guiding the robot near an object to be moved)}
+	\penaltyitem[5]{-30}{Receiving human help (telling robot which category an object is or where to place it)}
+	\penaltyitem[5]{-60}{Receiving human help (physically interacting with object)}
+
+	\scoreheading{Bonus rewards}
+	\scoreitem{200}{Opening entrance door without human help}
+	\scoreitem{150}{Moving a \emph{tiny} object}
+	\scoreitem{150}{Moving a \emph{heavy} object}
```

Comment:
whoever is running the test will also need to check that not only the number of objects in total is the same for all teams, but also that the number of `tiny` and `heavy` objects are the same for all teams.